enterprise/server/cmd/goinit/main.go

```diff
@@ -229,7 +235,12 @@ func main() {
 	die(mkdirp("/mnt/dev", 0755))
 	die(mount("/dev", "/mnt/dev", "", syscall.MS_MOVE, ""))
 
+	// TODO(bduffany): Spawn vmvfs via the init binary like we do for vmexec, to
+	// save scratch disk space. I tried it, but for some reason the fuse library
+	// panics saying that "/dev/null" doesn't exist:
+	// https://github.com/hanwen/go-fuse/blob/915cf5413cdef5370ae3f953f8eb4cd9ac176d5c/splice/splice.go#L57
 	die(copyFile("/vmvfs", "/mnt/vmvfs", 0555))
+	die(copyFile("/init", "/mnt/init", 0555))
```

Comment:
Where is this used?

Reply:
This is so that we can re-exec the init process with `--vmexec` after we've chroot'ed.